build.sbt

```diff
@@ -778,7 +760,7 @@ lazy val scalaDist = Project("scala-dist", file(".") / "target" / "scala-dist-di
     ),
     publishArtifact in (Compile, packageSrc) := false
   )
-  .dependsOn(libraryAll, compiler, scalap)
+  .dependsOn(library, reflect, compiler, scalap)
```

Comment:
> Does this remove swing and parser combinators from the default repl

yes

as for making it easier again to access them, I would think that would happen in the context of https://github.com/scala/scala-dev/issues/326

> Does this remove scala-xml from the default repl

no

```
~/scala.213/build/pack % bin/scala
Welcome to Scala 2.13.0-20170321-002852-96c02a3 (Java HotSpot(TM) 64-Bit Server VM, Java 1.8.0_112).
Type in expressions for evaluation. Or try :help.

scala> <i>foo</i>
res0: scala.xml.Elem = <i>foo</i>
```